packages/data-stores/src/launch/actions.ts

```diff
@@ -52,11 +55,27 @@ export const setDomainSearch = ( domainSearch: string ) =>
 		domainSearch,
 	} as const );
 
-export const setPlanProductId = ( planProductId: number | undefined ) =>
+const __setBillingPeriod = ( billingPeriod: Plans.PlanBillingPeriod ) =>
 	( {
+		type: 'SET_PLAN_BILLING_PERIOD',
+		billingPeriod,
+	} as const );
+
+export const setPlanProductId = function* setPlanProductId( planProductId: number | undefined ) {
```

Comment:
No need to repeat `setPlanProductId` twice. It can be rewritten to

```suggestion
export function* setPlanProductId( planProductId: number | undefined ) {
```

or, to keep consistency with the rest of the exported functions, to

```suggestion
export const setPlanProductId = function* ( planProductId: number | undefined ) {
```